pkg/framework/framework.go

```diff
@@ -136,7 +136,10 @@ func InitControllerHub(cc *kubeCl.CustomClient) (*ControllerHub, error) {
 		return nil, err
 	}
 	// Initialize Tekton controller
-	tektonController := tekton.NewSuiteController(cc)
+	tektonController, err := tekton.NewSuiteController(cc)
```

Comment:
maybe it looks better if change `tekton.NewSuiteController(cc)`  to `tekton.NewTekonController(cc)`

Reply:
I suggest keeping it "tekton.NewSuiteController(cc)" to keep it consistent with other controllers. Wdyt?